client/gutenberg/extensions/map-block/add-point/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+import { Component } from '@wordpress/element';
+
+import {
+	Button,
+	Dashicon,
+	Popover
+} from '@wordpress/components';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import LocationSearch from '../location-search';
+
+import './style.scss';
+
+export class AddPoint extends Component {
+
+	constructor() {
+		super( ...arguments );
+		this.onAddPoint = this.onAddPoint.bind( this );
+	}
+
+	onAddPoint( point ) {
+
+		this.props.onAddPoint( point );
+
+	}
+
+	render() {
+		const {
+			onClose,
+			onAddPoint
+		} = this.props;
+		return (
+			<Button
+				className='map__add_btn'
+			>
+				Add point
+				<Popover className='map__popover'>
+					<Button
+						className='map__popover_close'
+						onClick={ onClose }
+					>
+						<Dashicon icon="no" />
+					</Button>
+					<LocationSearch
+						onAddPoint={ onAddPoint }
+						label={ __( 'Add a location' ) }
+					/>
+				</Popover>
+			</Button>
+		);
+	}
+
+}
+
+AddPoint.defaultProps = {
+	onAddPoint: () => {},
```

Comment:
Use lodash's `noop` instead of defining empty functions?

Reply:
I'm happy to, but two notes on why I did this. 1) I pulled the pattern from https://github.com/WordPress/gutenberg/blob/master/packages/components/src/form-token-field/index.js#L601. 2) In an [earlier comment](https://github.com/Automattic/wp-calypso/pull/27836#discussion_r226453721) in this PR, @georgestephanis had suggested I take steps to remove `lodash` dependencies, but this would be adding one. Given these, do you still like this change? I'm more than happy to go either way, just asking about consistency.

Reply:
Ah, if our goal is to reduce `lodash` dependencies, then let's leave it as-is 👍